.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Add a new link to an existing icon
+    url: https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#adding-an-icon-to-lawnicons
+    about: Add new links via pull requests.
```

I think we could shorten the title text, and extend the about part. Here's an example:

```suggestion
  - name: Link apps to existing icons
    url: https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md#adding-an-icon-to-lawnicons
    about: Learn more about linking an app to an existing icon and making a PR to contribute to Lawnicons.
```

Feel free to change the text to something more suitable.